man/docker.1.md

```diff
@@ -26,125 +30,20 @@ To see the man page for a command run **man docker <command>**.
 **--help**
   Print usage statement
 
-**--api-cors-header**=""
-  Set CORS headers in the remote API. Default is cors disabled. Give urls like "http://foo, http://bar, ...". Give "*" to allow all.
-
-**-b**, **--bridge**=""
-  Attach containers to a pre\-existing network bridge; use 'none' to disable container networking
-
-**--bip**=""
-  Use the provided CIDR notation address for the dynamically created bridge (docker0); Mutually exclusive of \-b
-
 **--config**=""
   Specifies the location of the Docker client configuration files. The default is '~/.docker'.
 
 **-D**, **--debug**=*true*|*false*
   Enable debug mode. Default is false.
 
-**-d**, **--daemon**=*true*|*false*
-  Enable daemon mode. Default is false.
-
-**--default-gateway**=""
-  IPv4 address of the container default gateway; this address must be part of the bridge subnet (which is defined by \-b or \--bip)
-
-**--default-gateway-v6**=""
-  IPv6 address of the container default gateway
-
-**--default-ulimit**=[]
-  Set default ulimits for containers.
-
-**--dns**=""
-  Force Docker to use specific DNS servers
-
-**--dns-search**=[]
-  DNS search domains to use.
-
-**-e**, **--exec-driver**=""
-  Force Docker to use specific exec driver. Default is `native`.
-
-**--exec-opt**=[]
-  Set exec driver options. See EXEC DRIVER OPTIONS.
-
-**--exec-root**=""
-  Path to use as the root of the Docker exec driver. Default is `/var/run/docker`.
-
-**--fixed-cidr**=""
-  IPv4 subnet for fixed IPs (e.g., 10.20.0.0/16); this subnet must be nested in the bridge subnet (which is defined by \-b or \-\-bip)
-
-**--fixed-cidr-v6**=""
-  IPv6 subnet for global IPv6 addresses (e.g., 2a00:1450::/64)
-
-**-G**, **--group**=""
-  Group to assign the unix socket specified by -H when running in daemon mode.
-  use '' (the empty string) to disable setting of a group. Default is `docker`.
-
-**-g**, **--graph**=""
-  Path to use as the root of the Docker runtime. Default is `/var/lib/docker`.
-
 **-H**, **--host**=[unix:///var/run/docker.sock]: tcp://[host:port] to bind or
 unix://[/path/to/socket] to use.
   The socket(s) to bind to in daemon mode specified using one or more
```

Comment:
should this part still be here, given that this describes the *daemon* address/socket to listen on?